styles.css

```diff
@@ -273,30 +291,19 @@ hr {
   background-color: #1c1a19;
   min-height: 100vh;
   display: flex;
+  margin-top: 30px;
   flex-direction: column;
 }
 
 .about .about-me {
   background-image: url(assets/Frame.png);
   background-repeat: no-repeat;
-  background-size: 40%;
   background-position: bottom right;
-  min-height: 80vh;
+  min-height: 100vh;
   display: flex;
   flex-direction: column;
 }
 
```

Comment:
- [ ] only tour about me section  consider increasing the left side margin so as to match the Figma design and this will help you even in your future job to deliver what you were supposed 
![marf](https://user-images.githubusercontent.com/60146030/220935888-449d3742-f8f7-493d-acef-ff918b5f16e5.PNG)